conf/modules.config

```diff
@@ -352,7 +352,8 @@ process {
         --alignInsertionFlush Right \
         --alignSplicedMateMapLminOverLmate 0 \
         --alignSplicedMateMapLmin 30 \
-        --chimOutType Junctions'
+        --chimOutType Junctions \
+        --quantMode GeneCounts'
```

Comment:
Nice! 🌟 maybe update the documentation to reflect this change? Possibly where the output files are described and where the options given to star are stated.

Then, should this file be added to stub for testing or not needed?

Just to make sure, this is directly added to the publish dir, right?

Reply:
Here is where the publishing happens: 
https://github.com/nf-core/modules/blob/61cd93ecc95f4988e79d3b773e70f2068fdef998/modules/nf-core/star/align/main.nf#L31

And here is where the stub is:
https://github.com/nf-core/modules/blob/61cd93ecc95f4988e79d3b773e70f2068fdef998/modules/nf-core/star/align/main.nf#L96
Testing is done here for the module:
https://github.com/nf-core/modules/tree/master/tests/modules/nf-core/star/align

Reply:
I agree it needs to be done, but I would do in one go for the whole pipeline, I added a line in https://github.com/nf-core/rnafusion/issues/368 to that effect. I think it would lead to a clearer documentation